bases/node/node.yaml

```diff
@@ -338,8 +338,6 @@ spec:
               mountPath: /config/server
             - name: store-stats
               mountPath: /config/store-stats
-            - name: ramdisk
-              mountPath: /tmp
           resources:
             limits:
```

Comment:
We don't get OOM error now after this change, but because of ephemeral-storage we do get contrainer restarts. I tried ephemeral-storage parameter with few numbers but actual limit was being applied was 10G from some other parameters. Because of that I am disabling state-sync for now until we can find a good solution for ephemeral-storage or something in between persistent storage and ephemeral-storage.